central/debug/service/service.go

```diff
@@ -60,8 +60,8 @@ const (
 
 	centralClusterPrefix = "_central-cluster"
 
-	metricsPullTimeout     = 10 * time.Second
-	diagnosticsPullTimeout = 10 * time.Second
+	metricsPullTimeout     = 20 * time.Second
```

Comment:
Does this really take 20 seconds to load data from a local http server?

Reply:
Now it has timeouts. Maybe the issue is somewhere else. But after couple of timeouts one diagnostic bundle is created correctly.